compose/integrations/composable-test-cases/build.gradle.kts

```diff
@@ -22,7 +20,9 @@ allprojects {
 
         maven("https://packages.jetbrains.team/maven/p/kt/dev")
         maven("https://redirector.kotlinlang.org/maven/dev")
-//        mavenLocal()
+        if (System.getenv("TEAMCITY_VERSION") != null) {
+            mavenLocal()
```

Comment:
What is the issue with current configuration?

Not sure why it helps, because the current configuration doesn't publish anything to Maven Local: https://teamcity.jetbrains.com/admin/editRunType.html?id=buildType:JetBrainsPublicProjects_Compose_IntegrationComposableTests&runnerId=RUNNER_1&cameFromUrl=%2Fadmin%2FeditBuildRunners.html%3Fid%3DbuildType%253AJetBrainsPublicProjects_Compose_IntegrationComposableTests%26init%3D1&cameFromTitle=

Reply:
it uses artifact dependency (dependencies tab in configuration) and artifacts from that dependency should be placed in mavenlocal folder

Reply:
Using Maven Local is usually not advised, it can contain a lot of extra libraries that are downloaded by some other task and cached there, including broken ones.

Alternatively, we can copy the library into `build/maven-project`, and use it this way:
```
flatDir {       
    dirs("build/maven-project")
}
```